Makefile

```diff
@@ -25,7 +25,8 @@ lint-bandit: ## Run bandit
 	@echo "\n${BLUE}Running bandit...${NC}\n"
 	@${POETRY_RUN} bandit -r ${PROJ}
 
-lint-base: lint-flake8 lint-bandit ## Just run the linters without autolinting
+#lint-base: lint-flake8 lint-bandit ## Just run the linters without autolinting
+lint-base: lint-flake8 # TODO: Can we drop bandit?
```

Comment:
@edmondop is there a compelling reason to include bandit in this project? I found it was giving irrelevant warnings and since this project doesn't deal in any non-public data, it's unclear to me why we care that much about security.